app/models/organisation.rb

```diff
@@ -81,9 +81,21 @@ def self.search_by_keyword(keyword)
      self.where(contains_description(keyword).or(contains_name(keyword)))
   end
 
-  def self.filter_by_category(category_id)
-    return all unless category_id.present?
-    self.joins(:categories).where(is_in_category(category_id)) #do we need to sanitize category_id?
+  def self.filter_by_categories(category_ids)
```

Comment:
We might consider rewriting this function as I've done here:
https://github.com/mtc2013/LocalSupport/commit/fab5b834d26f33cb40b09a5d62691c1c90b36425

Incidentally, I think this form would make it very easy to switch to an AND logic simply by substituting arel and for the 'or'.